### PR TITLE
vsc-config path changes (HPC-7386)

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by wdpypere** [comment]
@itkovian tests fail

**Comment by itkovian** [comment]
@wdpypere yes, because we need a newer vsc-config and that isn't there yet :(

File: setup.py [hunk 1]
```diff
@@ -24,15 +24,15 @@
...
+    'version': '1.6',
     'author': [ag, jt],
     'maintainer': [ag, jt],
     'tests_require': ['mock'],
     'makesetupcfg': False,  # use setup.cfg provided to get pytz instead of python-pytz
     'install_requires': [
         'vsc-accountpage-clients >= 0.9.1',
         'vsc-base >= 2.8.3',
-        'vsc-config >= 1.35.3',
+        'vsc-config >= 1.48.0',
```
Comment (kwaegema) [inline setup.py]: > 2.0 ?

**Comment by kwaegema** [comment]
@itkovian I'll merge when tested on ces03 then :)

File: setup.py [hunk 2]
```diff
@@ -24,15 +24,15 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '1.5.2',
+    'version': '1.6',
```
Comment (stdweird) [inline setup.py]: @itkovian please don't change numbering scheme every time.

Comment (itkovian) [reply setup.py]: OK, I clearly forgot the .0.

**Pull request merged**